db/migrate/20190909231415_create_player_table.rb

```diff
@@ -0,0 +1,7 @@
+class CreatePlayerTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :players do |t|
+      t.string :username
+    end
+  end
+end
```

Comment:
```suggestion

end
```
Super nitpick, but some issues can occasionally arise if the last line is not a new line, this becomes more important when working on large teams where people are on different operating systems. (This applies to pretty much every file)
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline